### PR TITLE
Fix compilation issues discovered by CI

### PR DESCRIPTION
**Pull request by Kakadu**
* bad checksum
* missing dependency

It would be great if compilation errors would be sent on my email upon discovery.

### CODE REVIEW

File: packages/qocamlbrowser/qocamlbrowser.0.2.9/opam [hunk 1]
```diff
@@ -11,14 +11,14 @@ remove: [
   [make "uninstall"]
 ]
 depends: [
-  "ocaml" {>= "4.04.0"}
+  "ocaml" { >= "4.04.0" & <= "4.06.1" }
```
[the remark this answers is not in the document]
Comment (Kakadu) [reply packages/qocamlbrowser/qocamlbrowser.0.2.9/opam]: Yes, it seems that <= 4.06 is too optimistic restriction

**Comment by Kakadu** [comment]
@mseri Do you know, the message like this one is my error or a temporary server error?

    Command "sh" "-c" "docker build --label com.docker.datakit.digest=cb6bc4d5855a1650b0a244be5c660190 --label com.docker.datakit.builton=1544441002.46 --pull --rm --force-rm - < /tmp/tmp-1a0f5b35/Dockerfile" exited with status 40

**Comment by mseri** [comment]
That was a CI problem. I have restarted it. Now it fails with
```
[ERROR] At /home/opam/.opam/repo/default/packages/lablqml/lablqml.0.5.2/opam:20:29:
        '.' is not a valid token
[ERROR] Strict mode: aborting
```

File: packages/qocamlbrowser/qocamlbrowser.0.2.10/opam [hunk 2]
```diff
@@ -18,12 +18,12 @@ remove: [
   [make "uninstall"]
 ]
 depends: [
-  "ocaml" {>= "4.04.0"}
+  "ocaml" {>= "4.07.0"}
```
Comment (kit-ty-kate) [inline packages/qocamlbrowser/qocamlbrowser.0.2.10/opam]: why this change? It seemed to work ok in https://github.com/ocaml/opam-repository/pull/12384

**Comment by kit-ty-kate** [comment]
`lablqt` also needs fixing. Check out: http://check.ocamllabs.io/?maintainers=Kakadu

I'm not against removing releases that cannot detect Qt versions correctly

**Comment by kit-ty-kate** [comment]
Is this PR still a WIP?

**Comment by Kakadu** [comment]
@kit-ty-kate yes

**Comment by Kakadu** [comment]
So, lablqml compiles fine on all distros.
QOcamlbrowser compiles on Ubuntu. On others there is an error during compilation (a file from my sources is not found but on Ubuntu it's OK, I have no idea what could be a problem now)
I propose to merge it as it is now.

**Comment by Kakadu** [comment]
Hmm, wait, I may found a solution of the last issue.....

**Comment by Kakadu** [comment]
No, I still can't figure out what is wrong. I installed Fedora to VM but it compiles fine there.

**Comment by mseri** [comment]
@Kakadu  can we cleanup the history and merge or you are still working on it?

**Comment by Kakadu** [comment]
I'm happy enough about thing as they currently are. Squash+merge should be decent result of this PR :)

**Comment by kit-ty-kate** [comment]
I'm not too happy with this PR as it makes two packages uninstallable

**Comment by Kakadu** [comment]
> I'm not too happy with this PR as it makes two packages uninstallable

Can you be more concrete?

**Comment by kit-ty-kate** [comment]
according to Camelus above:
```
these releases are not installable anymore (2): qocamlbrowser.0.2.7 qocamlbrowser.0.2.8
```

**Comment by kit-ty-kate** [comment]
`qocamlbrowser` 0.2.7 and 0.2.8 don't seem to be able to compile with OCaml 4.03 and 4.04 respectively

**Comment by mseri** [comment]
For 0.2.8 seems to be an issue with the qt version selected

**Comment by Kakadu** [comment]
qocamlbrowser above 0.2.7 is just updating sources for changed AST in the compiler. I would like to remove old versions and have only the latest one (I doubt somebody would care --- nobody reported me bugs about it). Will it be OK?

**Comment by avsm** [comment]
This one seems fine to merge -- it just needs a rebase. Any chance of resolving the conflict and pushing a merge @Kakadu ?  Sorry for letting the PR linger so long.

**Comment by Kakadu** [comment]
Currently it doesn't compile under Mac (Qt is not found and I don't know how to add OS-specific PATH variables). If it is OK, then it can be squashed and merged. @avsm

File: packages/qocamlbrowser/qocamlbrowser.0.2.10/opam [hunk 3]
```diff
@@ -23,8 +24,9 @@ depends: [
   "conf-qt" {>= "5.2.1"}
   "ocamlbuild" {build}
 ]
-synopsis: "OCamlBrowser clone written with OCaml and QtQuick 2."
 url {
-  src: "https://github.com/Kakadu/QOcamlBrowser_quick/archive/0.2.10.tar.gz"
-  checksum: "md5=c6ccac614f4f6f27cdd6ea1f45926fa9"
+  #src: "https://github.com/Kakadu/QOcamlBrowser_quick/archive/0.2.10.tar.gz"
+  src: "https://github.com/Kakadu/QOcamlBrowser_quick/archive/master.zip"
```
Comment (kit-ty-kate) [inline packages/qocamlbrowser/qocamlbrowser.0.2.10/opam]: why is this pointing to master?
